include/ttmlir/Conversion/TTIRToLinAlg/TTIRToLinAlg.h

```diff
@@ -0,0 +1,24 @@
+// SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef TTMLIR_CONVERSION_TTIRTOLINALG_TTIRTOLINALG_H
+#define TTMLIR_CONVERSION_TTIRTOLINALG_TTIRTOLINALG_H
+
+#include "mlir/Pass/Pass.h"
+#include "mlir/Transforms/DialectConversion.h"
+
+#include "mlir/Dialect/Linalg/IR/Linalg.h"
+#include "mlir/Dialect/Linalg/IR/LinalgInterfaces.h"
+#include "mlir/Dialect/Tensor/IR/Tensor.h"
+
+namespace mlir::tt {
+
+void populateTTIRToLinAlgPatterns(MLIRContext *ctx, RewritePatternSet &patterns,
```

Comment:
I think we should use `Linalg` instead of `LinAlg`. I know it's 2 words, but if you search the llvm repo for `linalg`, there's no `LinAlg` anywhere in use in the first 2 pages of results.

Reply:
True, I find this a little harder to parse myself, but that does seem more consistent 👍